toolchain/sem_ir/file.h

```diff
@@ -304,8 +298,23 @@ class File : public Printable<File> {
   auto set_has_errors(bool has_errors) -> void { has_errors_ = has_errors; }
 
  private:
+  // Allocates a copy of the given data using our slab allocator.
+  template <typename T>
+  llvm::MutableArrayRef<T> AllocateCopy(llvm::ArrayRef<T> data) {
+    // We're not going to run a destructor, so ensure that's OK.
+    static_assert(std::is_trivially_destructible_v<T>);
+
+    T* storage = static_cast<T*>(
+        allocator_.Allocate(data.size() * sizeof(T), alignof(T)));
+    std::uninitialized_copy(data.begin(), data.end(), storage);
+    return llvm::MutableArrayRef<T>(storage, data.size());
+  }
+
   bool has_errors_ = false;
 
+  // Slab allocator, used to allocate node blocks.
```

Comment:
"node blocks" -> "node and type blocks", or just "vectors storing MutableArrayRef".

Reply:
Done.